Lib/test/test_msvcrt.py

```diff
@@ -115,6 +115,50 @@ def test_heap_min(self):
         except OSError:
             pass
 
+    def test_GetErrorMode(self):
+        msvcrt.GetErrorMode()
+
+    def test_SetErrorMode(self):
+        old = msvcrt.GetErrorMode()
+        self.addCleanup(msvcrt.SetErrorMode, old)
+
+        returned = msvcrt.SetErrorMode(0)
+        self.assertIs(type(returned), int)
+        self.assertEqual(old, returned)
+
+    @unittest.skipUnless(Py_DEBUG, "only available under debug build")
+    def test_set_error_mode(self):
+        old = msvcrt.set_error_mode(msvcrt.REPORT_ERRMODE)
+        self.addCleanup(msvcrt.set_error_mode, old)
+
+        returned = msvcrt.set_error_mode(msvcrt.OUT_TO_STDERR)
+        self.assertIs(type(returned), int)
+        self.assertNotEqual(returned, -1)
+        self.assertEqual(old, returned)
+
+    @unittest.skipUnless(Py_DEBUG, "only available under debug build")
+    def test_CrtSetReportMode(self):
+        old = msvcrt.CrtSetReportMode(msvcrt.CRT_WARN,
+                                      msvcrt.CRTDBG_REPORT_MODE)
+        self.addCleanup(msvcrt.CrtSetReportMode, msvcrt.CRT_WARN, old)
+
+        returned = msvcrt.CrtSetReportMode(msvcrt.CRT_WARN,
+                                           msvcrt.CRTDBG_MODE_DEBUG)
+        self.assertIs(type(returned), int)
+        self.assertNotEqual(returned, -1)
+        self.assertEqual(old, returned)
```

Comment:
Same as above.

Please test also with extreme values for `reportType` and `reportMode`.

Reply:
Same as `test_set_error_mode` above.